
        
09:07:50 PM: DEBUG [conexion.py:23] Conexión exitosa: <connection object at 0x7fec4f8a4d50; dsn: 'user=oldoc password=xxx dbname=pyapp host=127.0.0.1 port=5432', closed: 0>
09:09:00 PM: DEBUG [conexion.py:23] Conexión exitosa: <connection object at 0x7f8973991d50; dsn: 'user=oldoc password=xxx dbname=pyapp host=127.0.0.1 port=5432', closed: 0>
09:09:00 PM: DEBUG [conexion.py:36] Se abrió correctamente el cursor: <cursor object at 0x7f8973228e50; closed: 0>
